packages/babel-plugin/src/__tests__/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import type {BabelFileResult} from '@babel/core';
+
+import * as babel from '@babel/core';
+import {describe, it, expect} from 'vitest';
+import plugin from '../index';
+import {PRAGMA} from '../identifiers.json';
+
+describe('Babel Plugin', ()=>{
+  it('correctly converts a function to string and back', ()=>{
+    const fn = /* js */ `
+      const t = ()=>null;
+
+      
+      ${PRAGMA}(({b: {z}} = {b: {z: null}})=>{ t(); return -151;});
```

Comment:
what is the PRAGMA business?

Reply:
We can just have a bunch of snapshot tests for this. It'll be a lot easier.

Reply:
I'll migrate to use fixtures